pkg/awx/connection.go

```diff
@@ -249,28 +250,26 @@ func (c *Connection) get(path string, query url.Values, output interface{}) erro
 func (c *Connection) rawGet(path string, query url.Values) (output []byte, err error) {
 	// Send the request:
 	address := c.makeUrl(path, query)
-	if glog.V(2) {
```

Comment:
don't you want to log the fact that request was sent even if you get an error?

Reply:
Note we still log before the http request is done in `client.Do(request)`
Creating the request object could fail on line 253, but that is a local error and would be returned and logged.